modules/@apostrophecms/rich-text-widget/ui/apos/components/AposTiptapImage.vue

```diff
@@ -1,26 +1,17 @@
 <template>
   <div class="apos-image-control">
-    <AposButton
-      type="rich-text"
-      :class="{ 'apos-is-active': buttonActive }"
-      :label="tool.label"
-      :icon-only="!!tool.icon"
-      :icon="tool.icon || false"
-      :icon-size="tool.iconSize || 16"
-      :modifiers="['no-border', 'no-motion']"
-      :tooltip="{
-        content: tool.label,
-        placement: 'top',
-        delay: 650
-      }"
-      @click="click"
-    />
-    <AposImageControlDialog
-      :active="active"
-      :editor="editor"
-      :has-selection="hasSelection"
-      @close="close"
-    />
+    <AposContextMenu
+      ref="contextMenu"
+      menu-placement="bottom-end"
+      :button="button"
+      :keep-open-under-modals="true"
```

Comment:
Use this prop to not set `data-apos-menu`. This allow in the click outside to keep the menu opened when another modal is opened above.
Needed for menus like image that can open another modal to select image and want to stay opened.